contentcuration/contentcuration/static/js/edit_channel/uploader/components/FormulasMenu/symbols.json

```diff
@@ -0,0 +1,237 @@
+[
+	{
+		"title": "Formulas",
+		"symbols": [
+			{"key": "^", "preview": "{a}^{b}", "title": "Superscript"},
```

Comment:
Is there a place where `title` is getting translated? If not, I think it should be okay for now, but would be good to open an issue to track it

Reply:
I've just added this to my list of follow-ups I've been working on - here's a corresponding entry
https://www.notion.so/learningequality/Exercise-creation-1a085a11eb494ccab3962a22c20dd202#bb016e7b769b42e19377f22bd72cfc47

The list is in progress and there will be more issues. I am not sure if/when to open a new GitHub issue or if Notion is enough - I think we can talk about that during our next sync?